builtin/providers/cloudflare/resource_cloudflare_record_test.go

```diff
@@ -119,11 +119,13 @@ func TestAccCloudFlareRecord_Updated(t *testing.T) {
 					testAccCheckCloudFlareRecordExists("cloudflare_record.foobar", &record),
 					testAccCheckCloudFlareRecordAttributesUpdated(&record),
 					resource.TestCheckResourceAttr(
-						"cloudflare_record.foobar", "name", "terraform"),
+						"cloudflare_record.foobar", "name", "terraform2"),
```

Comment:
Doesn't look like this is doing what I hoped - it seems to be checking the terraform state, instead of hitting the cloudflare API and checking the record there.